app/code/Magento/BundleImportExport/README.md

```diff
@@ -1,2 +1,27 @@
-Magento_BundleImportExport module implements Bundle products import/export functionality.
-This module is designed to extend existing functionality of Magento_CatalogImportExport module by adding new product type.
+# Magento_BundleImportExport module
+
+**Magento_BundleImportExport** module enables you to operate implements bundle products import/export functionality with bundle products.
```

Comment:
"to operate implements" sounds awkward, as I see it was recommended to replace "implements" word with "operate"